test/base_policy_override_actions_test.rb

```diff
@@ -74,11 +73,13 @@ def view
     user = User.create!(name: "Aaron")
     policy = OverrideActionsUserPolicy.new(user)
     policy.destroy?.must_equal true
+    policy.can?(:destroy).must_equal true
 
     # User not named Aaron gets normal privileges
     user = User.create!(name: "Jim")
     policy = OverrideActionsUserPolicy.new(user)
     policy.destroy?.must_equal false
+    #policy.can?(:destroy).must_equal false
```

Comment:
Should be able to uncomment this line, eh?

Reply:
🤦‍♂️